requirements.txt

```diff
@@ -1,4 +1,4 @@
-langchain>=0.0.205
+langchain>=0.0.281
```

Comment:
0.0.281 has the latest LiteLLM updates to langchain: 
https://github.com/langchain-ai/langchain/releases